apps/analytics/src/scripts/5_initial_participant_course_analytics.py

```
@@ -0,0 +1,60 @@
+db.connect()
+
+# Script settings
+verbose = False
+
+
+# find all courses that started in the past
+df_courses = get_running_past_courses(db)
+
+# iterate over all courses and compute the participant course analytics
+for idx, course in df_courses.iterrows():
+    print(f"Processing course", idx, "of", len(df_courses), "with id", course["id"])
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)